battery-widget/README.md

```diff
@@ -32,6 +32,7 @@ It is possible to customize widget by providing a table with all or some of the
 | `warning_msg_text` | _Battery is dying_ | Text of the warning popup |
 | `warning_msg_position` | `bottom_right` | Position of the warning popup |
 | `warning_msg_icon` | ~/.config/awesome/awesome-wm-widgets/battery-widget/spaceman.jpg | Icon of the warning popup |
+| `disable_battery_warning` | `false` | Disable low battery warning |
```

Comment:
Could you please invert this option. I know it sounds like not a big deal, but I really believe that `enable_battery_warning` would be better. Also the double negation here ("disable" together with "false") might sound a bit misleading. 
I would really appreciate it.

Reply:
enable_battery_warning and the default being... true?

Reply:
Yep

Reply:
Updated the PR